alpha/hal.cpp

```diff
@@ -272,6 +272,17 @@ void _TS_HAL::rtc_get(TS_DateTime &dt)
   EXIT_CRITICAL;
 }
 
+uint32_t _TS_HAL::rtc_get_mins(void)
```

Comment:
this should go into cleanbox if it doesn't already exist

Reply:
into cleanbox.h?

Reply:
yeah